src/test/kotlin/com/terraformation/backend/accelerator/db/ParticipantProjectSpeciesStoreTest.kt

```diff
@@ -102,4 +104,55 @@ class ParticipantProjectSpeciesStoreTest : DatabaseTest(), RunsAsUser {
           "Participant Project Species IDs")
     }
   }
+
+  @Nested
+  inner class Create {
+    @Test
+    fun `creates the entity with the supplied fields`() {
+      val participantId = insertParticipant()
+      val projectId = insertProject(participantId = participantId)
+      val speciesId = insertSpecies()
+
+      every { user.canCreateParticipantProjectSpecies(projectId) } returns true
+
+      val participantProjectSpecies =
+          store.create(
+              NewParticipantProjectSpeciesModel(
+                  feedback = "feedback",
+                  id = null,
+                  projectId = projectId,
+                  rationale = "rationale",
+                  speciesId = speciesId,
+                  submissionStatus = SubmissionStatus.NotSubmitted))
+
+      assertEquals(
+          ParticipantProjectSpeciesRow(
+              feedback = "feedback",
+              id = participantProjectSpecies.id,
+              projectId = projectId,
+              rationale = "rationale",
+              speciesId = speciesId,
+              submissionStatusId = SubmissionStatus.NotSubmitted),
+          participantProjectSpeciesDao.fetchOneById(participantProjectSpecies.id))
+    }
+
+    @Test
+    fun `does not create the species association if the project is not associated to a participant`() {
+      val projectId = insertProject()
+      val speciesId = insertSpecies()
+
+      every { user.canCreateParticipantProjectSpecies(projectId) } returns true
+
+      assertThrows<ProjectNotInParticipantException> {
+        store.create(
+            NewParticipantProjectSpeciesModel(
+                feedback = "feedback",
+                id = null,
+                projectId = projectId,
+                rationale = "rationale",
+                speciesId = speciesId,
+                submissionStatus = SubmissionStatus.NotSubmitted))
+      }
+    }
+  }
```

Comment:
For completeness' sake, could use a test case for throwing an exception if the user doesn't have the right permission.